### PR TITLE
Voicekit - nice simple example to say IP address

### PR DESCRIPTION
**Pull request by sheridat**
This is my first PR - not sure if I have done it properly
Added an additional function to the assistant_library_with_local_commands_demo.py that retrieves the IP address and says it back to the user.

Note that the trigger phrase is "your address" - tried a few variants with IP in the phrase but that typically results in the assistant telling you your public rather than local address.

### CODE REVIEW

**Comment by t1m0thyj** [comment]
Glad to see that someone made a PR for this, and fixed the code that gets the IP address to make it work with both Raspbian Jessie and Stretch 👍 

Personally, I think it would make sense if the "power off" and "reboot" commands required the word "raspberry" before them like they did in the old master branch. Then it would make sense to have the command "raspberry IP address", which would probably be distinct enough from a personal address.

**Comment by sheridat** [comment]
@t1m0thyj  Hi, I tried numerous variants of trigger phrases.  For example if I try "Raspberry IP address" or "Raspberry Pi Address" the Google Assistant goes and fetches from "learn.adafruit.com" a how to find your IP address fragment.  More than happy for the google team to change it to what they want.

**Comment by divx118** [comment]
@sheridat Your file is in the wrong place. It is in the main dir of your git. You should just git clone your forked git to your local pc edit the file and then use use git push to your forked git. After that you can create a pull request. 
Best practice is also to create your own branch a duplicate of the voicekit and make your changes in that. It will then be easier to keep the original forked branches up to date when other pull requests get merged before yours.

File: assistant_library_with_local_commands_demo.py [hunk 1]
```diff
@@ -0,0 +1,100 @@
...
+	aiy.audio.say('Good bye!')
+	subprocess.call('sudo shutdown now', shell=True)
+
+
+def reboot_pi():
+	aiy.audio.say('See you in a bit!')
+	subprocess.call('sudo reboot', shell=True)
+
+def your_ip_address():
+        shell_command = "ip route get 1 | awk '{print $NF;exit}'"
```
Comment (divx118) [inline assistant_library_with_local_commands_demo.py]: This doesn't work all the time. The ip address is not always the last entry. Something like `ip route get 1 | awk -F 'src ' '{ split($2,a," ");print a[1];exit}'`should solve that.

Comment (sheridat) [reply assistant_library_with_local_commands_demo.py]: why don't we just let everybody do ifconfig?

File: assistant_library_with_local_commands_demo.py [hunk 2]
```diff
@@ -0,0 +1,100 @@
...
+        output = subprocess.getoutput(shell_command)
+        aiy.audio.say(output)
```
Comment (divx118) [inline assistant_library_with_local_commands_demo.py]: Maybe it is an option to point out that audio.say can take 2 argumants, so using ie  `aiy.audio.say(output,lang="en-US")`

**Comment by sheridat** [comment]
Hi, I'm happy to just kill this - people can just do ifconfig if they need to find out their pi's local ip address

**Comment by sheridat** [comment]
I keep trying to close this

**Comment by sheridat** [comment]
How do I delete this Pull Request?

**Comment by sheridat** [comment]
I'm happy enough to close this pull request - unfortunately just as I didn't know how to create it I haven't got a clue regarding how to close it.

**Comment by drigz** [comment]
Hey @sheridat! Thanks for giving this a try. If you'd like to get it merged, let me know and I'll give you some pointers on how to reopen and clean it up.

By the way - you did a better job than I did on [my first pull request](https://github.com/drigz/python-qrcode/pull/1), where I accidentally sent it to myself, got confused, gave up and didn't try again until the year after...

**Pull request closed without merging**